src/GlobalErrorBoundary.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import * as branding from './branding'
+import { msg } from './intl'
+import './error_styles-nomodules.css'
+import AppConfiguration from './config'
+
+class GlobalErrorBoundary extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = { hasError: false }
+    this.props.errorBridge.setErrorHandler((error) => {
+      this.setState({ hasError: true })
+      console.error(error)
+    })
+  }
+
+  componentDidCatch (error, info) {
+    this.setState({ hasError: true })
+    console.error(error)
+    console.error(info.componentStack)
+  }
+
+  render () {
+    const title = msg.globalErrorBoundaryTitle()
+    const descr = msg.globalErrorBoundaryDescription()
+    const gitHub = msg.gitHub()
+    const logOut = msg.logOut()
+    const refresh = msg.refresh()
+    const refreshUrl = AppConfiguration.applicationURL
+    const logoutUrl = AppConfiguration.applicationURL + '/sso/logout'
+    if (this.state.hasError) {
+      return (
+        <div>
+          <nav className='navbar obrand_mastheadBackground obrand_topBorder navbar-pf-vertical'>
+            <div className='navbar-header'>
+              <a href='/' className='navbar-brand obrand_headerLogoLink' id='pageheader-logo'>
+                <img className='obrand_mastheadLogo' src={branding.resourcesUrls.clearGif} />
+              </a>
+            </div>
+          </nav>
+          <div className='container text-center globalErrorContainer'>
```

Comment:
Use the `style['globalErrorContainer']` form as I mentioned above.  This will maintain consistency with the rest of the code and prevent CSS class name collisions.